src/release/breaking-changes/flutter-gradle-plugin-apply.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Deprecated imperative apply of Flutter Gradle Plugin
+description: >
+  How to migrate your Android app's Gradle build files to the new, declarative
+  format.
+---
+
+## Summary
+
+To build a Flutter app for Android, the Flutter Gradle plugin must be applied.
+Historically, this was done imperatively with Gradle's
+[legacy, imperative apply method][].
+
+In Flutter 3.16, support has been added for applying these plugins with Gradle's
+[declarative plugins {} block][], and it is the recommended approach. Since
+Flutter 3.16, projects generated with `flutter create` use the new apply method
```

Comment:
Agreed.

[Gradle itself refers](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block) to `plugins {}` as "`plugins {}` block", so my proposal is:
- imperative _apply script method_ is the old way of applying Gradle plugins
- _declarative plugins {} block_ is the new way of applying Gradle plugins

I don't want to bikeshed over this too much so I'll let you make the final call and I'm fine with whatever.

Reply:
I like the delta between the 2 you listed. I agree on wanting to avoid bikeshedding. 

* imperative apply script method is the old way of applying Gradle plugins
* declarative plugins {} block is the new way of applying Gradle plugins

Reply:
done in a7b28e38ff8998e0d75b0244dddce8c2361039cb